docs/docs/concepts/deployment.md

```diff
@@ -0,0 +1,84 @@
+# Deployment
+
+Once you build your LangGraph agent, you then have to think about deploying it.
+How you deploy your applications depends a bit on what it is and how you intend to use it.
+For simple applications, it can be pretty easy to deploy LangGraph.
+As your applications get more complicated (more nodes, longer running) or rely more on persistence (conversation memory, long-term memory, human-in-the-loop) then deployment can get trickier.
+In this guide, we go over a few of the considerations you will want to take into account when deploying more complicated LangGraph applications.
+
+> 📘 LangGraph Platform
+>
+> This guide covers concepts related to deployment in general. [LangGraph Platform](./langgraph_platform.md) is an opinionated way to deploy LangGraph agents built by the LangChain team, and handles all the issues raised here.
+
+## Simple deployment
+
+If your application is simple, then it is quite easy to deploy LangGraph agents - you can deploy them as you would any other Python application. For example, you could [FastAPI](https://fastapi.tiangolo.com/) (or an equivalent framework) to deploy your LangGraph application.
+
+```python
+# main.py
+
+from fastapi import FastAPI
+from your_agent_package import graph
+
+app = FastAPI()
+
+@app.get("/foo")
+async def root():
+    return await graph.ainvoke({...})
+```
+
+For simple applications, this is sufficient and is not that different from deploying Python applications in general, and so we will not go into this in too much detail here.
+
+For more complicated agent deployments (whether using LangGraph or not), you will run into some issues. The rest of this guide will discuss those issues.
+
+> 📘 LangGraph Platform
+>
+> This guide covers concepts related to deployment in general. [LangGraph Platform](./langgraph_platform.md) is an opinionated way to deploy LangGraph agents built by the LangChain team, and handles all the issues raised here.
+
+## Streaming support
+
+[Streaming](streaming.md) is a key part of LLM applications. It is usually necessary to stream back both tokens and intermediate steps of longer running agents. This is because LLM applications can take a while to run end-to-end, and so streaming helps show the user that something is happening.
+
+LangGraph support [multiple different streaming modes](streaming.md). When deploying, you will want to make sure that you expose the relevant endpoints.
```

Comment:
> When deploying, you will want to make sure that you expose the relevant endpoints.

do we talk about this anywhere? i.e. as a first-time user i might not know what the "relevant endpoints" are in this case